pkg/cmd/cli/cmd/config.go

```diff
@@ -46,6 +46,7 @@ func NewCmdConfig(parentName, name string) *cobra.Command {
 	cmd.Short = "Change configuration files for the client"
 	cmd.Long = configLong
 	cmd.Example = fmt.Sprintf(configExample, parentName, name)
+	cmd.Aliases = []string{"configs"}
```

Comment:
I'm not as sure about this one.  It mirrors a kubectl command that isn't aliased.